### PR TITLE
[KAN-549] Updating Referral Amount For US and Canada

### PR DESCRIPTION
**Pull request by Sonyque**
USA and Canada will have $ 50 refferal amount rather than $ 100.

### CODE REVIEW

File: src/regions.ts [hunk 1]
```diff
@@ -1522,6 +1522,7 @@ export const regions: BrandRegions = {
       currencyCode: "KRW",
       currencyPrefix: "",
       flagId: "kr_mrnua8",
+      flightsSupportEmail: DEFAULT_FLIGHTS_SUPPORT_EMAIL,
```
Comment (nguyenchr) [inline src/regions.ts:1525]: this change seems unrelated?

Comment (Sonyque) [reply src/regions.ts:1525]: Yeah, absence of `flightsSupportEmail` property brakes type script interface. So I decided to fix it.
